packages/core/admin/admin/src/content-manager/components/BlocksInput/BlocksInput.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+
+import { Box, Flex, Typography } from '@strapi/design-system';
+import { type MessageDescriptor, useIntl } from 'react-intl';
+import styled from 'styled-components';
+
+// @ts-expect-error TODO convert to typescript
+import { Hint } from '../Hint';
+
+import { BlocksEditor } from './BlocksEditor';
+
+const TypographyAsterisk = styled(Typography)`
+  line-height: 0;
+`;
```

Comment:
this can be done inline via a prop, although you probably need to do `"0px"` to stop it trying to use the theme value